futures-util/src/sink/mod.rs

```diff
@@ -9,7 +9,7 @@ use super::future::Either;
 use core::mem::PinMut;
 
 mod close;
-mod fanout;
+// mod fanout;
```

Comment:
Why is `fanout` commented out?

Reply:
Because the tests fail. There is no more `iter_ok()` for instance. Fixing now...

Reply:
No this is a rabbit hole... it requires converting `future_util/src/stream/forwards.rs` and probably various other things

Reply:
Can you leave `fanout`, but disable the tests? I'd like to make sure I know what code still needs fixing to work on 0.3 (regardless of tests) and what still needs fixing

Reply:
Done. Tests are disabled now.